README.md

```diff
@@ -48,19 +48,38 @@ git clone https://github.com/builderio/gpt-crawler
 npm i
 ```
 
+or 
+
+```sh
+pnpm i
+```
+
 #### Configure the crawler
 
 Open [config.ts](config.ts) and edit the `url` and `selector` properties to match your needs.
 
 E.g. to crawl the Builder.io docs to make our custom GPT you can use:
+This way will generate a unique json everytime!
 
 ```ts
+import { Config } from "./src/config";
+
+// protocol
+let protocol = "https://www.";
+// Important staff for eg: https://www.builder.io/c/docs/**
+let domain = "builder";
+let tld = ".io";
+// url
+let extra = "/c/docs/developers";
+// match
+let content = "/c/docs";
+let rest = "/**";
```

Comment:
i'm not sure this is beneficial - it is quite verbose. I think the way configuration is currently handled is intuitive and easy for individuals to structure in any way they want

Reply:
> i'm not sure this is beneficial - it is quite verbose. I think the way configuration is currently handled is intuitive and easy for individuals to structure in any way they want

The idea is that as user I cannot see difference between the two links, this may help me to improve the tool on a great way! <3 Thanks for your time, by the way!

Reply:
And Merry Christmas @steve8708